### DISCUSSION TITLE
b站个人空间视频专栏的视频无法倒序播放，建议增加正反序播放

### DISCUSSION THREAD

**Discussion by lyjhyjyes** [the1812/Bilibili-Evolved#2368] [功能建议 / Ideas]
韩风talk的KODI应该这么玩系列，在个人主页https://space.bilibili.com/472755599/channel/detail?cid=98008&ctype=0 可选默认排序和倒序排序，但发现两者都不是按照时间顺序排序。
请问能否增加按照时间顺序排序的功能？同时增加根据时间顺序在播放页面正反序播放功能
目前倒序排序（默认排序）后，点击播放全部，的确是倒序排序播放，可是在播放页面不能实现正反序的切换，只能顺序播放、随机播放、单个循环。

**Comment by lyjhyjyes** [comment]
> KODI的百变皮肤，不了解一下？ https://www.bilibili.com/medialist/play/472755599?business=space_channel&business_id=98008&desc=1 韩风talk的KODI应该这么玩系列，在个人主页https://space.bilibili.com/472755599/channel/detail?cid=98008&ctype=0 可选倒序排序。倒序排序后，点击播放全部，的确是倒序排序播放，可是在播放页面不能实现正反序的切换，只能顺序播放、随机播放、单个循环。

请问在个人空间搜索“kodi”关键词，https://space.bilibili.com/472755599/search/video?keyword=kodi，能否增加全部播放，或者勾选部分视频播放的呢

**Reply by Geeyun-JY3** [reply]
据我所知用户自己创建播放列表的方式目前就只有收藏视频，搜索关键词然后全部播放或勾选部分视频直接播放一播放列表不太可能，曲线救国如先创建一个收藏夹不知道是否可行。

**Comment by Geeyun-JY3** [comment]
视频列表（原频道）里视频的顺序，应该是 UP 自己排的，我觉得你应该让 UP 去调，这应该是一个以 UP 主为中心的功能。实在等不及，建议自己用收藏夹排。

**Reply by lyjhyjyes** [reply]
> 视频列表（原频道）里视频的顺序，应该是 UP 自己排的，我觉得你应该让 UP 去调，这应该是一个以 UP 主为中心的功能。实在等不及，建议自己用收藏夹排。

发现evolved作者新增了该功能，感谢作者

**Reply by the1812** [reply]
我没加过这功能啊?

**Reply by Geeyun-JY3** [reply]
打的 label 可是 `feat: rejected`……

**Reply by lyjhyjyes** [reply]
> 我没加过这功能啊?

那应该是b站自带功能，我看错了。收藏播放不能按顺序太麻烦

**Reply by Geeyun-JY3** [reply]
> 那应该是b站自带功能，我看错了。

现在好像并没有自带你这里提的按照时间顺序排序这个功能。

> 收藏播放不能按顺序太麻烦

收藏默认的列表循环按的就是你自己排的顺序
![image](https://user-images.githubusercontent.com/49396421/137842272-8873728c-4de4-46f4-81e9-e305cb4f9c74.png)
